components/apps_form/apps_form_field/apps_form_field.tsx

```diff
@@ -13,7 +13,7 @@ import {displayUsername} from 'mattermost-redux/utils/user_utils';
 import GenericUserProvider from 'components/suggestion/generic_user_provider.jsx';
 import GenericChannelProvider from 'components/suggestion/generic_channel_provider.jsx';
 
-import TextSetting, {InputTypes} from 'components/widgets/settings/text_setting';
+import TextSetting, {InputTypes} from 'components/widgets/settings/text_setting'; 
```

Comment:
unnecessary change and most probably failing lint test

Reply:
How can i correct it , by making another PR request